components/application-broker/internal/broker/testing/listers.go

```diff
@@ -0,0 +1,64 @@
+package testing
+
+import (
+	"github.com/pkg/errors"
+
+	fakeistioclientset "istio.io/client-go/pkg/clientset/versioned/fake"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	fakekubeclientset "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/tools/cache"
+
+	fakeeventingclientset "knative.dev/eventing/pkg/client/clientset/versioned/fake"
+	"knative.dev/pkg/reconciler/testing"
+)
+
+var clientSetSchemes = []func(*runtime.Scheme) error{
+	fakekubeclientset.AddToScheme,
+	fakeeventingclientset.AddToScheme,
+	fakeistioclientset.AddToScheme,
+}
+
+type Listers struct {
+	sorter testing.ObjectSorter
+}
+
+func NewScheme() *runtime.Scheme {
+	scheme := runtime.NewScheme()
+
+	for _, addTo := range clientSetSchemes {
+		if err := addTo(scheme); err != nil {
+			_ = errors.Wrapf(err, "error while adding to scheme")
```

Comment:
now the NewScheme() method would fail silently if error occurs and return nil, which shouldn't happen. As this code is part of testing package, you could use panic here. This way you won't have to modify the whole method to return error. See:
https://github.com/kyma-project/kyma/blob/c24dec578c54fa35a8a9278d19e48b2621688536/components/event-bus/internal/knative/subscription/controller/testing/listers.go#L59-L68